mat64/dense.go

```diff
@@ -273,6 +297,9 @@ func (m *Dense) Reset() {
 	m.mat.Data = m.mat.Data[:0]
 }
 
+// Clone makes a copy of a into the receiver, overwriting the previous value of the receiver.
```

Comment:
Yeah probably this could be more forceful. Take some more of the text from interface doc and say that it's a duplication rather than a copy.